src/main/resources/application.properties

```diff
@@ -6,6 +6,7 @@ spring.datasource.driver-class-name=${datasource.formplayer.driverClassName}
 spring.datasource.url=${datasource.formplayer.url}
 spring.datasource.username=${datasource.formplayer.username}
 spring.datasource.password=${datasource.formplayer.password}
+spring.jpa.hibernate.ddl-auto=validate
```

Comment:
this validates the the Java Entity objects match with the DB schema

Reply:
(note this won't have any effect currently since we have it set to 'null' in local config: https://github.com/dimagi/commcare-cloud/blob/master/src/commcare_cloud/ansible/roles/formplayer/templates/application.properties.j2#L32)